src/main/java/com/playtika/automation/school/test/framework/clients/AuthFeignClient.java

```diff
@@ -0,0 +1,31 @@
+package com.playtika.automation.school.test.framework.clients;
+
+import com.playtika.automation.school.test.framework.configs.FromUrlEncodedClientConfiguration;
+import com.playtika.automation.school.test.framework.pojo.requests.AuthenticationRequest;
+import com.playtika.automation.school.test.framework.pojo.requests.RegistrationRequest;
+import com.playtika.automation.school.test.framework.pojo.responses.LoginResponse;
+import com.playtika.automation.school.test.framework.pojo.responses.RegistrationResponse;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@FeignClient(
+        name = "login-client",
+        url = "https://taschool-notes-service.herokuapp.com",
```

Comment:
should be placed in application properties

Reply:
fixed